Dockerfile

```diff
@@ -41,18 +46,15 @@ RUN mkdir /root/trainer/gin/datasets
 COPY ./trainer/gin/datasets/tfrecord.gin /root/trainer/gin/datasets/tfrecord.gin
 COPY ./trainer/gin/datasets/base.gin /root/trainer/gin/datasets/base.gin
 
-RUN mkdir /root/tmp
-RUN mkdir /root/tmp/ddsp
+# RUN mkdir /root/tmp
+# RUN mkdir /root/tmp/ddsp
```

Comment:
With version control it doesn't make sense to keep code comments.

Reply:
Done